docker/install

```diff
@@ -27,6 +27,7 @@ apt-get autoremove -y
 
 # put back 2 needed awscli packages
 apt-get install -y python-six python-colorama
+pip install awscli --ignore-installed six
```

Comment:
Is having the aws-cli installed on all cores necessary?

Reply:
Yes, else we will get a Python 'module not found' error in the core.